package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "webpack",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "jest"
```

Comment:
- [ ] Please kindly adjust the `start` script to be able to run the webpack development server installed. At the moment, when you run `npm start` it does not do anything.